CHANGELOG.md

```diff
@@ -48,6 +48,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 - The `remote status` command will now print the username, realname, and email
   of the logged-in user, if available.
+- Adding `--sharens` mode for `apptainer exec/run/shell`, which enables to run multiple
+  apptainer instances in the same user namespace.
```

Comment:
Please extend this to indicate that they all need to be run by the same parent process.  I believe it also shares the image, such that if anybody runs the second command with a different container image, the second one will be ignored.  I suggest explaining that in this release message.

Reply:
thanks! updated in the latest commit.